src/intl/en/page-what-is-ethereum.json

```diff
@@ -96,5 +96,16 @@
   "page-what-is-ethereum-the-year-in-ethereum-2021-desc": "Jan 17, 2022 - Josh Stark and Evan Van Ness",
   "page-what-is-ethereum-atoms-institutions-blockchains": "Atoms, Institutions, Blockchains",
   "page-what-is-ethereum-atoms-institutions-blockchains-desc": "- Why blockchains matter?",
-  "page-what-is-ethereum-ethereum-in-numbers-title": "Ethereum in numbers"
+  "page-what-is-ethereum-ethereum-in-numbers-title": "Ethereum in numbers",
+  "adoption-chart-column-2010-label": "2010",
```

Comment:
Do we need to translate years? cc: @lukassim

Reply:
Context:

![Screenshot 2023-01-06 at 09 59 27](https://user-images.githubusercontent.com/62268199/210977499-cb1b4de4-273a-4431-9c20-d1f1d9ce48c5.png)

Reply:
I think we can leave the years untranslated.
Having quickly looked through the top 5 non-Latin languages in Crowdin, all of them use Latin numbers for dates.

Reply:
I would use the Intl NumberFormat function to style numbers according to locale
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat
They shouldn't need to be translated.